packages/core/src/blockchain/txpool.ts

```diff
@@ -238,20 +238,21 @@ export class TxPool {
         onApplyExtrinsicError: (extrinsic, error) => {
           this.event.emit(APPLY_EXTRINSIC_ERROR, [extrinsic, error])
         },
-        onPhaseApplied: logger.isLevelEnabled('trace')
-          ? (phase, resp) => {
-              switch (phase) {
-                case 'initialize':
-                  logger.trace(truncate(resp.storageDiff), 'Initialize block')
-                  break
-                case 'finalize':
-                  logger.trace(truncate(resp.storageDiff), 'Finalize block')
-                  break
-                default:
-                  logger.trace(truncate(resp.storageDiff), `Apply extrinsic ${phase}`)
+        onPhaseApplied:
+          logger.level.toLowerCase() === 'trace'
```

Comment:
that's fine for trace, but not for other logging levels. still need to figure out why `isLevelEnabled` is not supported on browser